robottelo/performance/perf_thread.py

```diff
@@ -0,0 +1,92 @@
+"""Test utilities for multi-threading programming"""
+from robottelo.performance.perf_candlepin import CandlepinPerfTest
+
+import logging
+import threading
+import time
+
+
+class PerfThread(threading.Thread):
+    def __init__(self, threadID, threadName, time_result_dict):
+        threading.Thread.__init__(self)
+        self.threadID = threadID
+        self.threadName = threadName
+        self.time_result_dict = time_result_dict
+        self.logger = logging.getLogger('robottelo')
```

Comment:
I think you can move this logging definition to the module as a constant:

```python
LOGGER = logging.getLogger(__name__)
```

This will create a new log called robottelo.performance.thread (considering that you have renamed the module) and will help you identify which messages are related to this module. Then you can assign self.logger = LOGGER.